app/policies/organization_policy.rb

```diff
@@ -0,0 +1,19 @@
+class OrganizationPolicy < ApplicationPolicy
+  alias_method :organization, :record
+
+  def index?
+    true
+  end
+
+  def show?
+    user&.superadmin? || user&.active?(organization)
```

Comment:
Just for me to understand, when do we have a `user` here and when not?

Reply:
Pundit uses the following to setup the `user` variable:

https://github.com/coopdevs/timeoverflow/blob/b31466a64952a42e7da427146916760f12f81da9/app/controllers/application_controller.rb#L73-L79

Sometimes there is no `current_user`, so we should check its presence at least in controllers (like `OrganizationsController`) that don't use `before_action :authenticate_user!`.